web_src/js/features/repo-issue.js

```diff
@@ -123,9 +123,12 @@ export function initRepoIssueSidebarList() {
       fullTextSearch: true,
     });
 
-  function excludeLabel(item) {
-    const href = $(item).attr('href');
-    const id = $(item).data('label-id');
+  /**
+   * @param {HTMLElement} item
+   */
+  function excludeLabel(item) { // eslint-disable-line unicorn/consistent-function-scoping
```

Comment:
Move the function to top-level and remove the comment :)

Reply:
[`1579b43` (#30083)](https://github.com/go-gitea/gitea/pull/30083/commits/1579b43671b6b1bddf53187c9c8ca6f837d63c51)